www/pages/docs/component-model.md

```diff
@@ -22,7 +23,7 @@ class FooterComponent extends HTMLElement {
   getTemplate() {
     const year = new Date().getFullYear();
 
-    return \`<header>This is the header component.  &copy; ${year}</header>\`;
+    return \`<header>This is the header component.  &copy; \${year}</header>\`;
```

Comment:
Is this expected, that now code blocks inside code fences need to be escaped?  If it's markdown, it should treat it all as a string, right?

Or Is it because this is getting processed as JavaScript instead?

Reply:
Yea it has to be escaped, no it's not treating it as a string, might have something to do with syntax highlighting.

Reply:
Would be cool to get something like GitHub flavored markdown
<img width="775" alt="Screen Shot 2020-04-28 at 7 42 29 PM" src="https://user-images.githubusercontent.com/895923/80548358-c01e7300-8988-11ea-88a9-22ace6545c5e.png">

<img width="763" alt="Screen Shot 2020-04-28 at 7 42 34 PM" src="https://user-images.githubusercontent.com/895923/80548367-c44a9080-8988-11ea-8cb9-ae521baa1560.png">